### ISSUE TITLE
skipped tabindex property breaks ModalDialog keydown event

### ISSUE THREAD

**Issue by zmousm** [videojs/video.js#6870]
## Description
`ModalDialog` modals can no longer be closed by pressing Escape. This happens because the `<div class=".vjs-modal-dialog">` element no longer gets an (explicit) `tabIndex` property after #6145 and it is thus not focusable.

https://codepen.io/zmousm/pen/bGpXror

## Steps to reproduce

1. Load and open any `ModalDialog`.
2. Press Escape.

### Expected
The modal should close.

### Actual
The modal does not close.

#### videojs
7.7.0 up to latest
#### browsers
any
#### OSes
any
